src/components/Chips/Chips.tsx

```diff
@@ -47,6 +46,18 @@ interface ChipsProps extends VibeComponentProps {
    * Callback function to be called when the user clicks the component.
    */
   onMouseDown?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  /**
+   * Callback function to be called when the user clicks the component.
+   */
+  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  /**
+   * Applies when element has onClick or onMouseDown props
+   */
+  ariaLabel?: string;
+  /**
+   * Should element be focusable & clickable - for backward compatability
+   */
+  isClickable?: boolean;
```

Comment:
Added in order not to break anything old, which is using chip - e.g. Dropdown with multi-state